app/Http/Controllers/Api/ContestsController.php

```diff
@@ -22,10 +25,17 @@ public function __construct()
     }
 
     /**
-     * Get a collection of waiting rooms.
+     * Get a collection of contests
      */
-    public function index()
+    public function index(Request $request)
     {
+        $run_nid = $request->query('run_nid');
```

Comment:
there is two lines under it using the variable? would you prefer i just did request->query('run_nid') twice?